server.ts

```diff
@@ -57,6 +57,12 @@ app.engine('html', ngExpressEngine({
 app.set('view engine', 'html');
 app.set('views', join(DIST_FOLDER, 'browser'));
 
+let xFrameOptions = require('x-frame-options');
+app.use(xFrameOptions());
+app.get('/', function (req, res) {
+  res.setHeader('X-Frame-Options', 'DENY');
+});
```

Comment:
I believe the `app.get()` part above can be removed as it's duplicating the implementation of `xFrameOptions`, see https://github.com/domharrington/x-frame-options/blob/master/index.js#L5